pyedflib/edfwriter.py

```diff
@@ -126,17 +126,22 @@ def isbytestr(s):
     return isinstance(s, bytes)
 
 
-def gender2int(gender):
-    if isinstance(gender, int) or gender is None:
-        return gender
-    elif gender.lower() in ['', 'x', 'xx', 'xxx', 'unknown', '?', '??']:
+def sex2int(sex):
+    if isinstance(sex, int) or sex is None:
+        return sex
+    elif sex.lower() in ['', 'x', 'xx', 'xxx', 'unknown', '?', '??']:
         return None
-    elif gender.lower() in ["female", "woman", "f", "w"]:
+    elif sex.lower() in ["female", "woman", "f", "w"]:
         return 0
-    elif gender.lower() in  ["male", "man", "m"]:
+    elif sex.lower() in  ["male", "man", "m"]:
         return 1
     else:
-        raise ValueError(f"Unknown gender: '{gender}'")
+        raise ValueError(f"Unknown sex: '{sex}'")
+
+
+def gender2int(sex):
+    warnings.warn('This function is deprecated.', DeprecationWarning, stacklevel=2)
+    sex2int(sex)
```

Comment:
```suggestion
    return sex2int(sex)
```

Reply:
That error is gone in the new version written from scratch :grin: